doc/rest-api.yaml

```diff
@@ -133,6 +133,22 @@ definitions:
     title: Cluster represents high-level information about a LXD cluster.
     type: object
     x-go-package: github.com/lxc/lxd/shared/api
+  ClusterCertificatesPost:
```

Comment:
Tiny nit, the normal commit message for this is `doc/rest-api: Refresh swagger YAML` (capitalization off in yours)